OBSDOCS-1664: Power monitoring Tech Preview 0.4 release notes (#87880)
openshift/openshift-docs: observability/power_monitoring/power-monitoring-release-notes.adoc

For an overview of the {PM-operator}, see xref:../power_monitoring/power-monitoring-overview.adoc#power-monitoring-about-power-monitoring_power-monitoring-overview[About {PM-shortname}].

[id="power-monitoring-release-notes-0-4"]
== {PM-shortname-c} 0.4 (Technology Preview)

This release includes the following version updates:

* {PM-kepler} 0.7.12
* {PM-operator} 0.15.0

[id="power-monitoring-0-4-features"]
=== Features

* With this release, FIPS mode is now enabled for {PM-title}. When installed on an {OCP} cluster in FIPS mode, {PM-operator} ensures compatibility without affecting the cluster’s FIPS support status.

> * Power monitoring is now migrated from Continuous Productization as a Service (CPasS) to Konflux. No action is required from customers.

Reviewer: @vimalk78 Do we need to mention this in the release notes? Isn't this internal to us?

> * Power monitoring is now migrated from Cloud Platform as a Service (CPasS) to Konflux improving platform stability, performance, and automation. No action is required from customers.

Reviewer 2: CPaaS is "Continuous Productization as a Service"

> * Power monitoring is now migrated from Cloud Platform as a Service (CPasS) to Konflux improving platform stability, performance, and automation. No action is required from customers.

Reviewer 2: the change from CPaaS to konflux is only in the way the product is built. this does not "platform stability, performance, and automation"

For an overview of the {PM-operator}, see xref:../power_monitoring/power-monitoring-overview.adoc#power-monitoring-about-power-monitoring_power-monitoring-overview[About {PM-shortname}].

[id="power-monitoring-release-notes-0-4"]
== {PM-shortname-c} 0.4 (Technology Preview)

This release includes the following version updates:

* {PM-kepler} 0.7.12
* {PM-operator} 0.15.0

> [id="power-monitoring-0-4-features"]

Reviewer 3: ```suggestion
[id="power-monitoring-0-4-features_{context}"]
```

=== Features

> * With this release, FIPS mode is now enabled for {PM-title}. When installed on an {product-title} cluster in FIPS mode, {PM-operator} ensures compatibility without affecting the cluster’s FIPS support status.

Reviewer 3: ```suggestion
* With this release, FIPS mode is now enabled for {PM-title}. When installed on an {product-title} cluster in FIPS mode, {PM-operator} ensures compatibility without affecting the FIPS support status of the cluster.
```

https://www.ibm.com/docs/en/ibm-style?topic=grammar-possessives

> Do not use possessive ’s with inanimate objects.

> [id="power-monitoring-bug-fix-0-4-features"]

Reviewer 3: ```suggestion
[id="power-monitoring-bug-fix-0-4-features_{context}"]
```

=== Bug fixes

> * Before this update, the *Install* screen in the Operator Hub page for {PM-title} displayed incorrect documentation links. With this update, the links now direct to the correct paths.

Reviewer 3: ```suggestion
* Before this update, the *Install* screen in the OperatorHub page for {PM-title} displayed incorrect documentation links. With this update, the links now direct to the correct paths.
```

> [id="power-monitoring-release-notes-0-4-CVEs"]

Reviewer 3: ```suggestion
[id="power-monitoring-release-notes-0-4-cves_{context}"]
```
